src/Sylius/Bundle/CoreBundle/Resources/config/app/workflow/sylius_order_checkout.yaml

```diff
@@ -0,0 +1,55 @@
+framework:
+    workflows:
+        !php/const Sylius\Component\Core\OrderCheckoutTransitions::GRAPH:
+            type: state_machine
+            marking_store:
+                type: method
+                property: checkoutState
+            supports:
+                - Sylius\Component\Core\Model\OrderInterface
+            initial_marking: !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_CART
+            places:
+                - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_CART
+                - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_ADDRESSED
+                - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_SHIPPING_SELECTED
+                - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_SHIPPING_SKIPPED
+                - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_PAYMENT_SELECTED
+                - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_PAYMENT_SKIPPED
+                - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_COMPLETED
+            transitions:
+                !php/const Sylius\Component\Core\OrderCheckoutTransitions::TRANSITION_ADDRESS:
+                    from: 
+                        - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_CART
+                        - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_ADDRESSED
+                        - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_SHIPPING_SELECTED
+                        - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_SHIPPING_SKIPPED
+                        - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_PAYMENT_SELECTED
+                        - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_PAYMENT_SKIPPED
+                    to: !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_ADDRESSED
+                !php/const Sylius\Component\Core\OrderCheckoutTransitions::TRANSITION_SKIP_SHIPPING:
+                    from:
+                        - !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_ADDRESSED
+                    to: !php/const Sylius\Component\Core\OrderCheckoutStates::STATE_SHIPPING_SKIPPED
+                !php/const Sylius\Component\Core\OrderCheckoutTransitions::TRANSITION_SELECT_SHIPPING:
+                    from:
```

Comment:
This new definition syntax needs to be backported to shipping (and possibly review) workflows
![image](https://github.com/Sylius/Sylius/assets/870747/c611e76d-a70e-44b0-b100-c8c199ace138)